css-device-adapt/Overview.bs

```diff
@@ -66,6 +63,63 @@ for a good discussion of these issues.
 Issue: Various issues about this specification and related specifications
 are listed in <a href="https://www.w3.org/Graphics/SVG/WG/wiki/Proposals/Investigation_of_APIs_for_Level_of_detail#The_issues_on_existing_APIs">this report</a>.
 
+<h2 id="scenarios">
+Motivating Scenarios</h3>
+
+<div class="example">
+    In this example, the document can be rendered without issue with any
+    size viewport. The author indicates this using the
+    <code class=html>&lt;meta&gt;</code> viewport tag.
+
+    <pre class="lang-html">
+        &lt;!doctype html>
+        &lt;html>
+        &lt;head>
+        &lt;meta name="viewport" content="width=device-width,initial-scale=1">
+        &lt;title>My Site&lt;/title>
+        &lt;/head>
+        &lt;body>
+            Since this document is just some simple text, it can be rendered
+            at any width without issue.  The text will just re-wrap as needed
+            when viewed in a smaller viewport.
+        &lt;/body>
+        &lt;/html>
+    </pre>
+</div>
```

Comment:
Same comment as the intro. Maybe this spec should normatively define the meta tag. But for now it doesn't, and I think we should resolve on that before adding examples that rely on it.